src/sidebar/util/test/is-third-party-service-test.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const proxyquire = require('proxyquire');
+
+describe('sidebar.util.isThirdPartyService', () => {
```

Comment:
We are being a bit inconsistent in how these describe blocks are named. I've been using the file path with slashes replaced by periods. You're using a mix of path and function name.

We have already standardised on one describe block per test module, so perhaps we should just make the label a relative path from the root (ie. `sidebar/util/is-third-party-service`). SGTY?

Reply:
I guess the confusion is between the module name `my-thing` vs the name of the actual function thing `myThing` or class `MyThing` (that is used both in the code in the module itself and in the assignment whenever anyone imports the module and then whenever anyone uses it).

If this were a module that contained more than one thing, I think you'd want to do:

```javascript
describe('sidebar/util/my-things', () => {
  describe('firstThing', () => {
  });
  describe('secondThing', () => {
  });
});
```

I guess that kind of implies that for a module containing one thing, we should do:

```
describe('sidebar/util/my-thing', () => {
  describe('myThing', () => {
  });
});
```